Extensions/TileMap/tilemapruntimeobject.js

```diff
@@ -0,0 +1,242 @@
+/**
+ * Displays a tilemap (mapeditor.org supported).
+ * @memberof gdjs
+ * @class TileMapRuntimeObject
+ * @extends RuntimeObject
+ */
+gdjs.TileMapRuntimeObject = function(runtimeScene, objectData) {
+  gdjs.RuntimeObject.call(this, runtimeScene, objectData);
+  this._frameElapsedTime = 0;
+
+  /** @type {number} */
+  this._opacity = objectData.content.opacity;
+  /** @type {string} */
+  this._tilemapJsonFile = objectData.content.tilemapJsonFile;
+  /** @type {string} */
+  this._tilemapAtlasImage = objectData.content.tilemapAtlasImage;
+  /** @type {string} */
+  this._displayMode = objectData.content.displayMode;
+  /** @type {number} */
+  this._layerIndex = objectData.content.layerIndex;
+  /** @type {number} */
+  this._animationSpeed = objectData.content.animationSpeed;
```

Comment:
Can you rename this to `_animationSpeedScale`/`animationSpeedScale`?

Scale is important here. This is not a speed, which units is a number of thing per seconds. Here, it's a number where 1 means "normal speed", 2 means "two times faster", 0.5 means "two times slower", etc...

The speed is defined by `animationFps`. `animationSpeedScale` is a multiplier.